index.js

```diff
@@ -95,6 +95,7 @@ app.post('/users/:userId/restaurants/visits', authentication.isAuthenticated, vi
 
 app.post('/follow', authentication.isAuthenticated, user.follow)
 app.delete('/follow/:id', authentication.isAuthenticated, user.unfollow)
+app.get('/follow/:id', authentication.isAuthenticated, user.findIfFollowed)
```

Comment:
@Elidoudly can we get the unsecure version as well please

Reply:
Consider it done captain